docs/1_exist_data_model.md

```diff
@@ -236,6 +236,46 @@ data = dict(train=dict(...), val=dict(...), test=dict(samples_per_gpu=2, ...))
 
 Or you can set it through `--cfg-options` as `--cfg-options data.test.samples_per_gpu=2`
 
+### Deprecate ImageToTensor
```

Comment:
Change the title to be Batch Inference, and add the following content as notice.